control-plane-operator/controllers/hostedcontrolplane/assets/olm/catalog-certified-operators.yaml

```diff
@@ -1,15 +0,0 @@
-apiVersion: "operators.coreos.com/v1alpha1"
```

Comment:
This seems to be undoing a change from a previous commit? Let's consolidate it in one atomic change with the expected result.